dir-locals.nix

```diff
@@ -1,4 +0,0 @@
-let pkgs = import <nixpkgs> {};
-    reflex-platform = import ./. {};
-in pkgs.nixBufferBuilders.withPackages
-     (builtins.filter (p: p != null) reflex-platform.tryReflexPackages)
```

Comment:
`git grep  dir-locals` turned up nothing

Reply:
This is for https://github.com/shlevy/nix-buffer, but I don't think anyone is actively using it.